### DISCUSSION TITLE
dbatools 2.0 delayed due to injury

### DISCUSSION THREAD

**Discussion by potatoqualitee** [dataplat/dbatools#8712] [Contributors]
My shoulder is super messed up and I will not be able to work on 2.0 for a bit. I return from vacation around the 10th and hope to start working on it again then (severe flareups usually last a couple weeks). I welcome any help on these https://github.com/dataplat/dbatools/issues?q=is%3Aissue+is%3Aopen+label%3A%22fix+in+2.0%22

When I get back, my first priority will be to create 2.0 beta in the gallery. If you need any help getting started, @wsmelton, @jpomfret and @sanderstad know how to get you working from the dev branch. maintainers with the nuget key are welcome to compile the dbatools library and publish, no need to sign yet. the compile scripts are in the dbatools library repo under build.

**Comment by jpomfret** [comment]
Sorry to hear that @potatoqualitee - hope you're on the mend soon!
